oss-cloud-backend/handler.js

```diff
@@ -28,7 +28,7 @@ module.exports.addContributor = async (event, context, callback) => {
   // check if request is valid
   try {
     // TODO create generic function for checking validity of a body
-    var body = JSON.parse(event.body);
+    let body = JSON.parse(event.body);
     // check if keys exist
     body.username;
```

Comment:
This line as well as following two can be removed or commented. There is one more occurrence of this type at line no. 105.

Reply:
Removed the three lines in the first occurrence. The second occurrence is already overwritten in another branch.